make_profiler/report/README.md

```diff
@@ -0,0 +1,12 @@
+# Makefile Profiler Report
+
+Reports large data processing pipeline steps.
+
+## Usage
+...   
+
+## Notes
+
+Uglifying code    
+npm i -g uglifyjs   
```

Comment:
there is no such package on npm as uglifyjs (https://www.npmjs.com/package/uglify-js)
```suggestion
npm i uglify-js -g 
```

Reply:
I think that it would be better to just drop uglification here, if we're not building a hi-performance frontend here it will work just fine and be more hackable with the non-uglified script.